File: exam_prep/level2/doop/doop.c
#include <string.h>
#include <stdlib.h>
#include <stdio.h>

int main(int argc, char **argv)
{
	int i;

	i = 0;
	if (argc == 4)
	{
		if (!argv[1] || !argv[2] || !argv[3])
			return (0);
		if (argv[2][0] == '+')
			printf("%d", (atoi(argv[1]) + atoi(argv[3])));
		else if (argv[2][0] == '-')
			printf("%d", (atoi(argv[1]) - atoi(argv[3])));
		else if (argv[2][0] == '/')
			printf("%d", (atoi(argv[1]) / atoi(argv[3])));
		else if (argv[2][0] == '%')
		printf("%d", (atoi(argv[1]) % atoi(argv[3])));
		else if (argv[2][0] == '*')
			printf("%d", ((atoi(argv[1])) * (atoi(argv[3]))));

	}
	printf("\n");
	return (0);
}

File: php/d01/ex10/do_op.php
#!/usr/bin/php
<?php

if ($argc == 4)
{
	$num1 = trim($argv[1]);
	$operator = trim($argv[2]);
	$num2 = trim($argv[3]);

	if ($operator == "+")
		$res = $num1 + $num2;
	else if ($operator == "-")
		$res = $num1 - $num2;
	else if ($operator == "/")
		$res = $num1 / $num2;
	else if ($operator == "%")
		$res = $num1 % $num2;
	else if ($operator == "*")
		$res = $num1 * $num2;
	print($res)."\n";

}
else
{
	print("Incorrect Parameters\n");
}

?>